From: Kir Kolyshkin <kir@openvz.org>
Date: Mon, 9 Jun 2008 10:38:13 +0000 (+0400)
Subject: etc/vz/vz.conf: document MODULES_DISABLED
X-Git-Url: http://git.openvz.org/?p=vzctl;a=commitdiff_plain;h=626325d26903279dbe64a966eed5199be4316aa0;hp=d5fba5eee206a84557c0654a7998edf133039e36

etc/vz/vz.conf: document MODULES_DISABLED

Original report and patch by Peter <pva@gentoo.org>

Fixes http://bugzilla.openvz.org/888
Signed-off-by: Kir Kolyshkin <kir@openvz.org>

---

--- etc/vz.conf	2008-07-14 06:02:01 +0000
+++ etc/vz.conf	2008-07-14 06:02:14 +0000
@@ -14,6 +14,9 @@
 DISK_QUOTA=yes
 VZFASTBOOT=no
 
+# Disable module loading. If set, vz initscript do not load any modules.
+#MODULES_DISABLED=yes
+
 # The name of the device whose ip address will be used as source ip for VE.
 # By default automatically assigned.
 #VE_ROUTE_SRC_DEV="eth0"

